Documentation/Home/SystemExtensions.rst

```diff
@@ -83,10 +85,18 @@ part of the core and extracted at some point in time.
      :ExtKey: linkvalidator
      :Comment:
 
+   - :Manual: `Lowlevel <https://docs.typo3.org/c/typo3/cms-lowlevel/master/en-us/>`_
+     :ExtKey: lowlevel
+     :Comment:
+
    - :Manual: `Recycler <https://docs.typo3.org/c/typo3/cms-recycler/master/en-us/>`_
      :ExtKey: recycler
      :Comment:
 
+   - :Manual: `Redirects <https://docs.typo3.org/c/typo3/cms-redirects/master/en-us/>`_
+     :ExtKey: redirects
+     :Comment: Documentation added in 11/2021
```

Comment:
Do we need this info? If one wants to know, he can look it up through various sources. (Same goes for `rte_ckeditor` below.)

Reply:
Ok. It is not absolutely necessary here and maybe we can remove the comments entirely. Will push another commit.